watchtower-plugin/src/main.rs

```diff
@@ -483,10 +510,7 @@ async fn on_commitment_revocation(
             state.add_pending_appointment(tower_id, &appointment);
 
             if status.is_temporary_unreachable() {
-                state
-                    .unreachable_towers
-                    .send((tower_id, appointment.locator))
-                    .unwrap();
+                send_to_retrier(&state, tower_id, appointment.locator);
```

Comment:
Just noticed :o
towers with sub issues (possibly being retried) don't get new revocation data.

Reply:
I'm not sure I follow. `send_to_retrier` does send fresh data to a retrier as long as:

- There is an active running retrier
- There is no retrier at all

So mainly the only reason why a retrier won't get new data is if it is idle